site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -2,4 +2,79 @@
 api: runtime
 ---
 
-<!-- Intentionally blank -->
+## Examples
+
+### Use [`getURL`][getURL] to add an extension image to a page {: #example-getURL }
+
+Extensions can access their own resources using relative paths, but in order for the extension's
```

Comment:
This is kind of a run-on sentence that rushes past some assumed knowledge. Consider something like:

Extensions can access their own resources using relative paths (like "images/logo.png") but web pages can't, because they are running in a separate namespace. To let web pages access these resources, you need to specify...

Reply:
Took a shot at rewriting this intro. Let me what you think.